src/containers/Charts/Line.tsx

```diff
@@ -0,0 +1,253 @@
+import React, { useEffect, useState } from "react";
+import { useMappedState, useDispatch } from "redux-react-hook";
+
+import LineChart from "../../components/charts/Line/Line";
+import TextField from "@material-ui/core/TextField";
+import InputLabel from "@material-ui/core/InputLabel";
+import MenuItem from "@material-ui/core/MenuItem";
+import FormControl from "@material-ui/core/FormControl";
+import Select from "@material-ui/core/Select";
+import Button from '@material-ui/core/Button';
+
+import { Blockchain, Block, State } from "../../types";
+import {
+    convertTimeStamp,
+    getSelectedDate
+} from "./helpers";
+
+const mapState = (state: State): Blockchain => ({
+    blokchain: state.tezos.blocks
+});
+
+const LineCharts = (): React.ReactElement => {
+    const dispatch = useDispatch();
+    const { blokchain } = useMappedState(mapState);
+    const [dateFrom, setDateFrom] = useState(getSelectedDate(7));
+    const [label, setLabel] = useState([
+        "19-04-2019",
+        "20-04-2019",
+        "21-04-2019",
+        "22-04-2019"
+    ]);
+    const [data, setData] = useState([10, 20, 30, 40]);
+    const [config, setConfig] = useState({
+        chartType: "buyers",
+        label: "Amount of the currency",
+        title: "Transactions for the selected buyer and day"
+    });
+    const [select, setSelect] = useState("buyer");
+    const [buyer, setBuyer] = useState("buyer");
+    const [seller, setSeller] = useState("xyz2");
+    const [chartType, setChartType] = useState("buyer");
+
+    const handleBuyerChange = (e: any) => {
+        setTimeout(() => setBuyer(e.target.value), 100)
+    };
+
+    const changeUserType = (e: any) => {
+        setChartType(e.target.value);
+    }
+
+    const filterChart = (blokchain: Block[], chartType: string): void => {
+        const labels: any[] = [];
+        let elements: number[] = [];
+
+        blokchain.forEach((item: Block): void => {
+            const timeStampConverted: string = convertTimeStamp(item.timestamp);
+
+            if (timeStampConverted === dateFrom) {
+                switch (chartType) {
+                    case "selers":
+                        if (item.source == seller) {
```

Comment:
Zawsze używaj "triple equals" zamiast dwóch, jak ustawimy eslinta jak trzeba to Ci na to nie pozwoli. Chyba że to celowo ale wtedy tym bardziej jest coś nie tak 😛jeśli jedna wartość jest zawsze stringiem, a druga number to wtedy trzeba parsowć żeby mieć kontrolę nad tym co porównujemy (jak zostawisz to JSowi to nikt nie wie czego się spodziewać 😄).